codespell_lib/data/dictionary_from_LO.txt

```diff
@@ -0,0 +1,3087 @@
+autocoomit->autocommit
+autofilt->autofilter
+autofomat->autoformat
+autogrowwith->autogrowwidth
+autoincrementive->autoincrement
+automatially->automatically
+automatical->automatically, automatic, automated,
+automtically->automatically
+autoutline->autooutline
+avaiblable->available
+availavle->available
+avalibale->available
+avaoid->avoid
+avarege->average
+avary->every
+avcoids->avoids
+aviod->avoid
+avtive->active
+awnser->answer
+axises->axes
+bacause->because
+bacic->basic
+backbround->background
+backsapce->backspace
+backtransparant->backtransparent
+backwords->backwards
+backwrad->backward
+bage->bag
+baisc->basic
+balacing->balancing
+ballonmode->balloonmode
+balse->false
+bandaid->band-aid
+barbedos->barbados
+basci->basic
+basf->base
+basicmanger->basicmanager
+basidide->basicide
+beasties->beasts
+beaviour->behaviour
+be be->be
+becasuse->because
+becauses->because
+bechmark->benchark
+beding->begging
+begginning->beginning
+begines->begins
+beginn->begin
+behabviour->behaviour
+behavious->behaviour
+behavioutr->behaviour
+behaviuor->behavior
+behinde->behind
+behing->behind
+behvaiour->behaviour
+beind->behind
+beinning->beginning
+bejond->beyond
+beliefes->beliefs, believes,
+belond->belong
+beloning->belonging
+benchamarked->benchmarked
+beneeth->beneath
+beseclass->baseclass
+betweeness->betweenness
+betwene->between
+beweeen->between
+bewteeness->betweenness
+beyound->beyond
+bginning->beginning
+billon->billion
+binairy->binary
+binanary->binary
+bistream->bitstream
+bitamps->bitmaps
+bizare->bizarre
+bizarely->bizarrely
+blak->blank
+blcok->block
+blcoks->blocks
+blokcer->blocker
+blured->blurred
+boddy->body
+bodgy->body
+bookmar->bookmark
+boold->bool
+booleanss->booleans
+boolen->boolean
+booolean->boolean
+bordrelines->borderline
+bordrelines->borderlines
+bothe->both
+botifies->notifies
+bottomborde->bottomborder
+botton->bottom
+boundaryi->boundary
+brach->branch
+brackeds->brackets
+brackground->background
+brance->branches
+braodcasted->broadcasted
+breakes->breaks
+brfore->before
+brigthness->brightness
+bringtofont->bringtofront
+broblematic->problematic
+bufer->buffer
+bufers->buffers
+buffereed->buffered
+bufferent->buffered
+buffor->buffer
+buffors->buffers
+bugous->bogus
+buguous->bugous
+buld->build
+bulle->bullet
+bult->built
+bult-in->built-in
+bumb->bump
+bumpded->bumped
+buttosn->buttons
+butttons->buttons
+buuild->build
+cach->catch
+cachse->cache
+caclate->calculate
+cahannel->channel
+cahce->cache
+calcable->calculable
+calcualtion->calculation
+calcualtions->calculations
+calculaion->calculation
+calcutated->calculated
+caleed->called
+caler->caller
+callef->called
+callibri->calibri
+calss->calls
+calsses->classes
+canadan->canadian
+cancelability->cancellability
+cancelles->cancels
+cancled->canceled
+cange->change
+cann't->can't
+caputre->capture
+caractere->character
+carfull->carefull
+carryintg->carrying
+caseingore->caseignore
+caseinlie->caseinline
+case-insenstive->case-insensitive
```

Comment:
wonder why this one is hyphenated where most of the other multi-words (see directly above and below) aren't

Reply:
This list if made of words found in the LO sources.

So, as I found 'em, I recorded.